src/resolvers/claims/status-card/statusCard.ts

```diff
@@ -0,0 +1,248 @@
+import { ResolverSubset } from '../../ResolverSubset'
+import {
+  ClaimStatusCard,
+  ClaimStatusCardPill,
+  ClaimStatusCardPillType,
+  ClaimStatusProgressSegment,
+  ClaimStatusProgressType,
+} from '../../../generated/graphql'
+import {
+  ClaimDto,
+  ClaimOutcomeDto,
+  ClaimStatusDto,
+} from '../../../api/upstreams/claimService'
+import { LocalizedStrings } from '../../../translations/LocalizedStrings'
+
+export const ClaimsStatusCardsSubset: ResolverSubset = {
+  Query: {
+    claims_statusCards: async (
+      _parent,
+      args: { locale: string },
+      { upstream, strings },
+    ): Promise<ClaimStatusCard[]> => {
+      const claims = (await upstream.claimService.getMemberClaims()).sort(
+        (a, b) =>
+          new Date(a.registrationDate).getTime() -
+          new Date(b.registrationDate).getTime(),
+      )
+      const localizedStrings = strings.withLocale(args.locale)
+
+      return Promise.all(
+        claims.map((c) =>
+          transformClaimStatusCard(c, args.locale, localizedStrings),
+        ),
+      )
+    },
+  },
+}
+
+// TODO This doesn't even need to be async right? Any reason we'd keep it async if it doesn't await for anything?
+const transformClaimStatusCard = async (
+  claim: ClaimDto,
+  locale: string,
+  strings: LocalizedStrings,
+): Promise<ClaimStatusCard> => {
+  return {
+    id: claim.id,
+    pills: transformClaimStatusPills(claim, locale, strings),
+    title: transformClaimTitle(claim, strings),
+    subtitle: transformClaimSubTitle(claim),
+    progressSegments: transformProgressSegments(claim, strings),
+  } as ClaimStatusCard
+}
```

Comment:
```suggestion
): Promise<ClaimStatusCard> => ({
    id: claim.id,
    pills: transformClaimStatusPills(claim, locale, strings),
    title: transformClaimTitle(claim, strings),
    subtitle: transformClaimSubTitle(claim),
    progressSegments: transformProgressSegments(claim, strings),
  })
```

Single-statement arrow functions in JS/TS need no curlies, in the same way as such functions work in Kotlin 🙂